### PR TITLE
integrate mypy into CI

### PR DESCRIPTION
**Pull request by aadibajpai**
reworked some stuff to make it pass

Signed-off-by: Aadi Bajpai <me@aadibajpai.me>

### CODE REVIEW

**Comment by aadibajpai** [comment]
> Hello @aadibajpai! Thanks for updating this PR.
> 
> * In the file [`swaglyrics_backend/issue_maker.py`](https://github.com/SwagLyrics/swaglyrics-backend/blob/2d9d2977ac4556706a06b2b8d4b2d1ff35eb1215/swaglyrics_backend/issue_maker.py):
> 
> > [Line 180:29](https://github.com/SwagLyrics/swaglyrics-backend/blob/2d9d2977ac4556706a06b2b8d4b2d1ff35eb1215/swaglyrics_backend/issue_maker.py#L180): [E701](https://duckduckgo.com/?q=pep8%20E701) multiple statements on one line (colon)
> > [Line 180:29](https://github.com/SwagLyrics/swaglyrics-backend/blob/2d9d2977ac4556706a06b2b8d4b2d1ff35eb1215/swaglyrics_backend/issue_maker.py#L180): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ':'
> > [Line 180:30](https://github.com/SwagLyrics/swaglyrics-backend/blob/2d9d2977ac4556706a06b2b8d4b2d1ff35eb1215/swaglyrics_backend/issue_maker.py#L180): [E225](https://duckduckgo.com/?q=pep8%20E225) missing whitespace around operator
> > [Line 180:30](https://github.com/SwagLyrics/swaglyrics-backend/blob/2d9d2977ac4556706a06b2b8d4b2d1ff35eb1215/swaglyrics_backend/issue_maker.py#L180): [E999](https://duckduckgo.com/?q=pep8%20E999) SyntaxError: invalid syntax

I don't think pep8speaks has support for py 3.8

**Comment by aadibajpai** [comment]
@thealphadollar, this PR does the same for backend (mypy in CI)

**Review by flabbet — APPROVED** [review]

File: mypy.ini [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[mypy]
+ignore_missing_imports = True
+warn_unused_ignores = True
```
Comment (thealphadollar) [inline mypy.ini]: Newline would be nice to have here.

**Review by thealphadollar — APPROVED** [review]
LGTM.

**Pull request merged**